docs/en/ingest-management/elastic-agent/configuration/examples/config-file-example-apache.asciidoc

```diff
@@ -0,0 +1,126 @@
+[[config-file-example-apache]]
+= Config file example: Apache HTTP Server
+
+++++
+<titleabbrev>Apache HTTP Server</titleabbrev>
+++++
+
+Include these settings in your standalone {agent} `elastic-agent.yml` configuration file to ingest data from Apache HTTP server. 
+
+* <<config-file-example-apache-logs>>
+* <<config-file-example-apache-metrics>>
+
+[[config-file-example-apache-logs]]
+== Apache HTTP Server logs
+
+["source","yaml"]
+----
+outputs: <1>
+  default:
+    type: elasticsearch <2>
+    hosts:
+      - 'https://94d343762a2b465f976aa292df6a0fe1.us-central1.gcp.cloud.es.io:443'
+    username: '${ES_USERNAME}'
+    password: '${ES_PASSWORD}'
+agent:
+  download: <3>
+    sourceURI: 'https://artifacts.elastic.co/downloads/'
+  monitoring: <4>
+    enabled: true
+    use_output: default
+    namespace: default
+    logs: true
+    metrics: true
+inputs: <5>
+  - id: logfile-apache-fb9e632d-3708-4e76-bf4e-9758d379507e
```

Comment:
@kilfoyle i believe this id here (and others) just needs to be unique. perhaps it's explained in the bats docs. I wouldn't want the reader to think that they have to insert this exact id here.

@cmacknz is that correct?

Reply:
Thanks. I've changed this and similar "ID" lines to `{user-defined-unique-id} <7>` and the description to `A user-defined unique ID to identify the input stream.`.